DigitalLearningSolutions.Web/appsettings.json

```diff
@@ -5,6 +5,8 @@
     "ReadOnlyConnection": "Data Source=localhost;Initial Catalog=mbdbx101_uar;Integrated Security=True;encrypt=false;TrustServerCertificate=true;",
     "UnitTestConnection": "Data Source=localhost;Initial Catalog=mbdbx101_uar_test;Integrated Security=True;encrypt=false;TrustServerCertificate=true;"
   },
+  "LearnerDefaultText": "<p>I confirm that I have completed the required pathway and achieved the necessary {{CompetencyAssessmentName}}. I understand that:</p><ul> <li>I am responsible and accountable for my practice</li><li>my skills and achievements are recorded for future reference</li><li>if I move to a new role or area, I may need to review or expand my skills</li><li>I must follow organisational policies and guidance</li><li>I should speak to a senior colleague if I am unsure about any aspect of my practice,</li></ul><p>my {{CompetencyAssessmentName}} will be reviewed as part of personal development and revalidation</p>",

```

Comment:
Since the value is the same for all environments. This should only be set here, in this file - the other environment appsettings files where it is set are unnecessary - they would only be used to override this with environment-specific values.